bokeh/palettes.py

```diff
@@ -1090,6 +1098,14 @@
 Colorblind4 = Colorblind8[:4]
 Colorblind3 = Colorblind8[:3]
 
+# Bokeh palette created from colors of shutter logo
+BokehPalette7 = ('#EC1557', '#F05223', '#F6A91B', '#A5CD39', '#20B254', '#00AAAE', '#892889')
+BokehPalette6 = BokehPalette7[:6]
+BokehPalette5 = BokehPalette7[:5]
+BokehPalette4 = BokehPalette7[:4]
+BokehPalette3 = BokehPalette7[:3]
```

Comment:
I think I'd prefer just `Bokeh` since everything in here is a palette, `BokehPalette` seems a bit redundant

Reply:
Fixed!